schemas/EGA.experiment.json

```diff
@@ -2,12 +2,324 @@
+            "properties": {
+              "assay_subtype": {
+                "anyOf": [
+                  {
+                    "title": "DNA-Assay subtype controlled vocabulary (CV) list",
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_dna.json"
+                  },
+                  {
+                    "title": "RNA-Assay subtype controlled vocabulary (CV) list",
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_rna.json"
+                  }
+                ]                 
+              }
+            }
+          },
+          {
+            "title": "Assay subtypes match array/sequencer assays",
+            "oneOf": [
+              {
+                "title": "Assay type and subtype terms are from the array CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_array.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_array.json" 
+                  }
+                }
+              },
+              {
+                "title": "Assay type and subtype terms are from the sequencer CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_sequencer.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_sequencer.json" 
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+
+      "assayed_molecule": {
+        "type": "string",
+        "title": "Assayed molecule of the experiment [EFO:0002772]",
+        "description": "The human readable ID (e.g. DNA assay), chosen from a list of controlled vocabulary (CV), of the assayed molecule. If you cannot find your term in the CV list, please create an issue at our [metadata GitHub repository](https://github.com/EbiEga/ega-metadata-schema) proposing its addition.",
+        "enum": ["DNA assay", "RNA assay", "Metabolomic profiling", "Protein assay"],
+        "meta:enum": ["DNA assay__EFO:0001456", "RNA assay__EFO:0001457", "Metabolomic profiling__EFO:0000752", "Protein assay__EFO:0001458"]
+      },
+
+      "types_of_data": {
+        "type": "array",
+        "title": "Types of data [IAO:0000100]",
+        "description": "Types of data the experiment encompasses.",
```

Comment:
```suggestion
        "description": "Types of data the experiment generates.",
```

Reply:
I think for experiment, you want the type of data the experiment produces? As opposed to the type of data an analysis uses as input? 

I'm thinking maybe this attribute is redundant with other attributes. For example, if you indicate `"assayed_molecule": "genomic DNA"` and relevant values for `assay_technology` and `assay_type_descriptor`, what value does indicating `"types of data": ["genomic data"]` add?

Reply:
Well, the molecule would not be ``genomic DNA`` but ``DNA`` alone. That way ``genomic data`` does add some information on whether the DNA is from a metagenomic conglomerate, synthetic DNA, or from a genome.

Reply:
Depending on what changes we apply to the assayed_molecule object my previous comment will make sense or not.